contributor-docs/operations/tests/rfc2119-keyword-test-guide.md

```diff
@@ -0,0 +1,157 @@
+---
+title: SCS RFC2119 Keyword Test Guide
+type:
+status: Draft
+track: Global
+---
+
+## Introduction
+
+The process of test creation and standards definition in Sovereign Cloud Stack (SCS)
+involves developing and validating tests that align with the specific requirements and
+standards set by SCS. This entails a comprehensive understanding of the SCS framework,
+which is built on open-source components and aims to provide interoperable and sovereign
+cloud services. In creating tests for SCS, it's essential to consider the various aspects
+of cloud technology, including infrastructure, services, and the integration of these
+components. The tests should be designed to ensure that the SCS standards are met, which
+includes adhering to specified requirement levels as indicated by [RFC2119 keywords](https://datatracker.ietf.org/doc/html/rfc2119).
+
+## 1. Understanding SCS Standards and RFC2119 Keywords
+
+SCS standards use RFC2119 keywords like:
+
+- **MUST** (same as **REQUIRED**, **SHALL**),
+- **MUST NOT** (same as **SHALL NOT**),
+- **SHOULD** (same as **RECOMMENDED**),
+- **SHOULD NOT** (same as **NOT RECOMMENDED**),
+- **MAY** (same as **OPTIONAL**)
+
+to define requirements for SCS-compatible IaaS and KaaS resources. These standards are crucial for ensuring interoperability and sovereignty in cloud services, as they are built on open-source components such as Kubernetes.
+
+To ensure readability and comprehensibility, only the main keywords **MUST**, **MUST NOT**, **SHOULD**, **SHOULD NOT** and **MAY** are referred to below.
+
+## 2. Interpreting RFC2119 Keywords in the Context of SCS
+
+- **MUST and MUST NOT**: These are binary and straightforward to test. Compliance or
+  non-compliance directly affects the standard meeting. For visualization, fulfilled
+  requirements could be marked in green, unfulfilled ones in red.
+- **SHOULD and SHOULD NOT**: These represent recommendations. While fulfilling these do
+  not directly affect standard compliance, it is advised for future-proofing against
+  potential standard evolutions. Non-compliance could be visualized in yellow, possibly
+  with a valid reason for non-fulfillment.
+- **MAY**: This is more flexible and could be used to provide additional
+  information or best practices. Their impact on compliance is not as direct as **MUST** or
+  **SHOULD** categories. These can be marked as blue.
+
+## 3. Channels for Output in Test Scripts
+
+- Align the RFC2119 keywords with specific channels in test scripts:
+  - **MUST** and **MUST NOT** lead to the ERROR channel for failed tests
+  - **SHOULD** and **SHOULD NOT** lead to the WARNING channel, the test is still passed
+  - **MAY** to leads to the INFO channel, the test is still passed
```

Comment:
The output channels seem to come out of "nowhere", have you considered introducing them briefly beforehand? And also why they are used (this might be good to know).

Something along the lines of 

> In test scripts, we use *channels* to convey information of different importance to the user. These channels are based on the ubiquitous and de-facto standard logging levels of common logging libraries in scripting and programming languages, in particular Python: DEBUG, INFO, WARNING, ERROR, and CRITICAL.

Furthermore, because the first enumeration level just has a single item ("- Align the RFC2119 keywords ..."), I think it should be "flattened".

Reply:
Added the explanation. 
Changed the enumeration level.